packages/gvim.rb

```diff
@@ -3,81 +3,50 @@
-            --enable-gui \
+            --localstatedir=#{CREW_PREFIX}/var/lib/vim \
+            --with-features=huge \
+            --with-compiledby='Chromebrew' \
+            --with-tlib=ncurses \
             --with-x \
+            --enable-gui \
             --enable-fontset \
             --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --enable-python3interp \
-            --enable-rubyinterp \
-            --with-python3-command=#{CREW_PREFIX}/bin/python3 \
-            --with-ruby-command=#{CREW_PREFIX}/bin/ruby \
-            --with-features=huge \
+            --enable-cscope \
+            --enable-perlinterp=dynamic \
+            --enable-pythoninterp=dynamic \
+            --enable-python3interp=dynamic \
+            --enable-rubyinterp=dynamic \
+            --disable-nls \
             --disable-selinux"
-    system 'make'
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81"
   end
 
   def self.install
-    tmpdir = "#{CREW_PREFIX}/tmp/vim"
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    configdestdir = "#{CREW_DEST_PREFIX}/.config/vim"
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    FileUtils.mkdir_p "#{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/evi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/*gvi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/evi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/*gvi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/vim/vim81/gvimrc_example.vim #{tmpdir}"
-    system "rm -rf #{CREW_DEST_PREFIX}"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/gvimrc_example.vim #{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/*.1 #{CREW_DEST_PREFIX}/share/man/man1"
-    system "mv #{tmpdir}/* #{CREW_DEST_PREFIX}/bin"
-    system "rm -rf #{tmpdir}"
+
     system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{configdestdir}"
-    FileUtils.mkdir_p "#{CREW_DEST_HOME}"
-    system "touch #{configdir}/gvimrc #{configdestdir}/gvimrc"
-    system "ln -s #{configdir}/gvimrc #{CREW_DEST_HOME}/.gvimrc"
-  end
 
-  def self.postinstall
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    system "cp #{CREW_PREFIX}/share/vim/vim81/gvimrc_example.vim #{configdir}/gvimrc" unless File.size? "#{configdir}/gvimrc"
-    puts
-    puts "To finish the installation, execute the following:".lightblue
-    puts "echo '# Fixes Warning: Missing charsets in String to FontSet conversion' >> ~/.bashrc".lightblue
-    puts "echo 'export LC_ALL=C' >> ~/.bashrc && . ~/.bashrc".lightblue
-    puts
-    puts "The config files are located in $HOME/.vim".lightblue
-    puts
+    # runtime files are provided by the vim package
+    system "rm", "-r", "#{CREW_DEST_PREFIX}/share/vim"
```

Comment:
No need to remove this folder since crew will automatically delete these working files after installation.

Reply:
This is still in `def self.install`. I think I know what he's doing. He's removing those files because they're provided by the `vim` package.

Reply:
Exactly, that's not obvious from how the diff looks, but that in `install`, I don't want the runtime files to end up in the binary package.

Reply:
I think we need to be careful here.  Are you sure the man files are vim and not gvim?

Reply:
The entire section you removed considered the overlap.

Reply:
The vim directory has `gvimrc_example.vim` and this will wipe that away.